### DISCUSSION TITLE
How do I turn off the logs that a module is installed?

### DISCUSSION THREAD

**Discussion by wishinghand** [vuejs/pinia#1723] [Help and Questions] [answered]
Every time a module in Pinia loads, there's a console.log consisting of:
`🍍 "{moduleName}" store installed `

How do I turn this off? Looking at the API in the docs, it's not clear if that option exists.

**Comment by wobsoriano** [comment, marked as answer]
Go to Vue Devtools and turn off "Notify about new/deleted stores"

<img width="500" alt="Screen Shot 2022-10-16 at 1 33 54 PM" src="https://user-images.githubusercontent.com/13049130/196056948-b3c001b2-c3cb-4516-a9db-95f3c2846197.png">

**Reply by KaliaJS** [reply]
@wobsoriano I don't have this option. Any idea why ? : 
<img width="906" alt="pinia-notify" src="https://user-images.githubusercontent.com/65311002/207878615-063b7767-a284-4e36-9a85-64dd112141f7.png">

**Comment by wishinghand** [comment]
Thanks! That did the trick.
